plone-and-python-community.rst

```diff
@@ -1819,7 +1818,7 @@ Plone Roadmap 2020
 
 .. pull-quote::
 
-    Plone stays the |br| First Class Citizen of |br| Content Management |br| and Python Web
+    Plone remains the |br| First Class Citizen of |br| Content Management |br| and Python Web
```

Comment:
maybe change "the" to "a" to avoid offending any Pyramid or Django etc. people in the audience?